packages/components/addon/components/hds/app-footer/legal-links.hbs

```diff
@@ -0,0 +1,22 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<Hds::AppFooter::Link
+  @href="https://www.hashicorp.com/support"
+  class="hds-app-footer__link--support"
```

Comment:
if these are intended only to be used as selector in the integration test, we should use a different selector (or a different approach?)

Reply:
I can look at these as part of clean up, thanks.

Reply:
I deleted these but now don't know how to test for them in integration tests